entries/c/cibc.com.json

```
@@ -5,14 +5,17 @@
     "tfa": [
       "sms",
       "call",
-      "email"
+      "custom-software"
     ],
-    "documentation": "https://www.cibc.com/en/privacy-security/two-step-verification-b.html",
-    "regions": [
-      "ca"
+    "custom-software": [
+      "CIBC App Push Notification"
     ],
+    "documentation": "https://www.cibc.com/en/privacy-security/two-step-verification-b.html",
```

This page this references email as a authentication method.

---

Indeed! The page referenced here is out-of-date. Here's the current one:

```suggestion
    "documentation": "https://www.cibc.com/en/privacy-security/two-step-verification.html",
```